pytac/load_csv.py

```diff
@@ -1,3 +1,5 @@
+""" Module to load the elements of the machine from a csv file."""
+
```

Comment:
It might be worth saying that you load machine data from multiple CSV files in a single directory.